artifactory_cleanup/cli.py

```diff
@@ -130,11 +147,27 @@ def main(self):
                 size(summary.artifacts_size),
             ]
             table.add_row(row)
+            output_json["policies"].append(
+                {
+                    "name": summary.policy_name,
+                    "file_count": summary.artifacts_removed,
+                    "size": size(summary.artifacts_size),
```

Comment:
Do we need to use `size` here? If we're going to create a way to pass data for the next script - it's better to pass bytes in `size`

What are you going to use output.json for, btw? Just curious